pages/[user].js

```diff
@@ -3,6 +3,7 @@ import { useRouter } from 'next/router';
 
 import { getIsomorphicDataPopulation } from '../store';
 import { loadUserPage } from '../store/actions';
+import UserFeedStatus from '../components/User-feed-status';
```

Comment:
The file should be called `UserFeedStatus`, following the style we use in other component files.